src/main/assembly/dist/examples/bag/agreements/example1.xml

```diff
@@ -22,12 +22,12 @@
             xmlns="http://easy.dans.knaw.nl/schemas/bag/metadata/agreements/"
 >
     <depositAgreement>
-        <signerId easy-account="user001">MisterX</signerId>
+        <signerId easy-account="user001" email="info@dans.knaw.nl">MisterX</signerId>
         <dcterms:dateAccepted>2018-03-22T21:43:01.000+01:00</dcterms:dateAccepted>
         <depositAgreementAccepted>true</depositAgreementAccepted>
     </depositAgreement>
     <personalDataStatement>
-        <signerId>MisterX</signerId>
+        <signerId easy-account="user001" email="info@dans.knaw.nl">MisterX</signerId>
```

Comment:
why is this information repeated? is that necessary?

Reply:
Do you mean the `<signerId>` or its attributes? For the latter:

Just in case at some point both elements somehow flow independently through some processing, you would not have to look for the other element to complete your data.